app/src/main/java/com/babylon/wallet/android/domain/seedphrase/SeedPhraseInputDelegate.kt

```diff
@@ -1,6 +1,8 @@
-package com.babylon.wallet.android.presentation.common
+package com.babylon.wallet.android.domain.seedphrase
```

Comment:
why you moved these delegates and model in the domain? 
This is presentation logic, especially when you have it UI stateful 🤷🏽‍♂️

Reply:
since it felt like those delegates contain only business logic

Reply:
imo functions and parameters like `onWordChanged` and `UiState` are pure UI logic and that's why this package with the delegates and the model makes more sense to be in presentation. 
Not a hard requirement from my side though.

Reply:
looking at structure of `presentation/` package, most subpackages have view model, screen file, nav file and some additional UI models if necessary. So those classes seemed to not fit into that "structure". If you want I can move them under `presentation.common.seedphrase`

Reply:
it's absolutely fine to have extra packages in presentation like common, utils or even better a `helper` package that contains all these UI-related delegates or other classes.
Up to you 👍🏽